### ISSUE TITLE
After any popup is closed, NVDA doesn't announce names of any buttons or links on the page and they can't be activated.

### ISSUE THREAD

**Issue by PetrashkoEV** [nvaccess/nvda#6394]
I created an example that simulates a pop-up opening. Please, take a look to the following example, where the issue fully reproducible.
[http://codepen.io/PetrashkoEV/pen/BLzRjE?editors=1010](http://codepen.io/PetrashkoEV/pen/BLzRjE?editors=1010)

**Steps to reproduce:**

1. Open this [page](http://codepen.io/PetrashkoEV/pen/BLzRjE?editors=1010)
2. Activate "Open popup button" or "Open popup span"
3. Activate "Close popup" button
4. Press TAB key for navigation to the first link and activate it by space or enter.

**Expected results:**
Open modal window

**Actual results:**
NVDA doesn't announce the name of the links and buttons. They are not activated after pressing Enter/Space button.

**Notes:**
This issue is reproduced in **Firefox 48**. In versions below 48 this issues isn't reproduced.
In my opinion, this is due to adding '**overflow-y**' attribute to 'html' tag

**Comment by jcsteh** [comment]
This is due to [Mozilla bug 1280551](https://bugzilla.mozilla.org/show_bug.cgi?id=1280551). I was literally just testing a fix for this and it seems to work.

I wish we'd had your test case months ago; your test case reproduces it 100% reliably, whereas other test cases we've been using are only able to reproduce it intermittently. :)
